kapacitor/templates/tickscript-init-job.yaml

```diff
@@ -0,0 +1,74 @@
+{{if .Values.install_tick_scripts.enabled }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}-init
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+data:
+  init.sh: |-
+    #!/usr/bin/env bash
+    export KAPACITOR_URL={{ .Values.install_tick_scripts.kapacitorURL }}
+    ls -al $TICK_DIR
+    kapacitor list tasks
+
+    echo "Loading TICK files from $TICK_DIR"
+    for f in ${TICK_DIR}/*.tick
+    do
+        name="$(basename $f)"
+        name="${name/.tick/}"
+        kapacitor define $name \
+            -type batch \
+            -dbrp telegraf.autogen \
```

Comment:
Do we have a better solution to hardcoding the type and dbrp of the task? In the past I have parsed the type and dbrp values from comments at the top of the TICKscript. Part of the work in https://github.com/influxdata/kapacitor/issues/1313 will be to standardize that behavior. 

For example
```
// kapacitor:type=stream
// kapacitor:dbrp=telegraf.autogen
```
 or 
```
// kapacitor:type=batch
// kapacitor:dbrp=mydatabase.autogen
```

Something along those lines is what I was thinking, but that will probably not be the final result of https://github.com/influxdata/kapacitor/issues/1313.

Technically it is possible to have multiple dbrps values but you can probably leave that out for now as it is uncommon.